server/lib/api/matchmakingTimes.ts

```diff
@@ -165,6 +166,10 @@ async function addNew(ctx: Koa.Context, next: Koa.Next) {
   const { startDate, enabled } = ctx.request.body as AddMatchmakingTimeBody
 
   ctx.body = await addMatchmakingTime(matchmakingType, new Date(startDate), !!enabled)
+
+  if (matchmakingStatusInstance) {
+    matchmakingStatusInstance.maybePublish(matchmakingType)
```

Comment:
Async function results need to be handled so that we don't end up with unhandled exceptions. Alternatively for here, since `maybePublish` is surrounded completely by a try/catch, you could expose a non-async version of that that calls through to a private `async` version but handles/logs errors from the `Promise` (you could also then remove the wrapping `try/catch` in that case since it would be redundant).